driver-reactive-streams/src/main/com/mongodb/reactivestreams/client/internal/VoidWriteOperationThenCursorReadOperation.java

```diff
@@ -39,12 +42,17 @@ public String getCommandName() {
     }
 
     @Override
-    public void executeAsync(final AsyncReadBinding binding, final SingleResultCallback<AsyncBatchCursor<T>> callback) {
-        writeOperation.executeAsync((AsyncWriteBinding) binding, (result, t) -> {
+    public BatchCursor<T> execute(final ReadBinding binding, final OperationContext operationContext) {
```

Comment:
No need to explicitly add this as it should be in the `ReadOperationCursorAsyncOnly` - lets update the default method there instead.